### ISSUE TITLE
helper executable is recompiled although it exists (windows)

### ISSUE THREAD

**Issue by jneira** [DanielG/cabal-helper#103]
Consider the output of vscode using the branch of haskell-language-server that uses cabal-helper: https://gist.github.com/jneira/3528613421538d221c1273271fa1ea89

It outputs three calls to the helper exe with the same output (including the hash):
```
using helper compiled with Cabal from v2-build package-env C:\Users\atrey\Local Settings\Cache\cabal-helper\ghc-8.6.5.package-envs\Cabal-3.2.0.0.package-env
helper exe does not exist, compiling C:\Users\atrey\Local Settings\Cache\cabal-helper\cabal-helper-1.0.0.0--Cabal-3.2.0.0--3df8897ef8e453832c4972293afda106b94b4bf0429a9688fa3066c3e36aa2dd.build\cabal-helper
....
```
The main difference is the first take 7.04515s to compile and the others 3.61770s.
I guess it is windows specific, but not sure.

**Comment by DanielG** [comment]
I don't really have much to go on here.. it says the exe doesn't exist, if that's true I'm not sure how we're supposed to conjure one out of thin air other than by compiling :)

Does the path change or something?

**Comment by jneira** [comment]
Ah, i forgot to mention a little detail 😄: the executable exists since the first time cabal-helper was executed, but with an `.exe` extension
So it is seem easy to fix at first glance

**Comment by jneira** [comment]
Closed by #117
